flang/test/Lower/intrinsic-procedures/RRSPACING.f90

```diff
@@ -0,0 +1,11 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
```

Comment:
combine with the spacing test below?

Reply:
But the generated intrinsic is `@_FortranARRSpacing16` 🤔

Reply:
Something is wrong here. A spacing test should not call RRSpacing. Running it outside, the call is to FortranASpacing and not FortranARRSpacing.

Reply:
I guess the fix is to call rrspacing in the test.

Reply:
Yes, but also these `! CHECK` lines are incorrect and ignored (missing `:`):  https://github.com/flang-compiler/f18-llvm-project/blob/fir-dev/flang/test/Lower/intrinsic-procedures.f90#L1116-L1117

 I'm just about to send an update.